### ISSUE TITLE
Don't show DEBUG message by default

### ISSUE THREAD

**Issue by jaklan** [klattimer/LGWebOSRemote#84]
DEBUG message should be visible only when run with e.g. `-v` flag, not always.

**Comment by klattimer** [comment]
#beta my friend :)

Will leave this open until it's got a pull request and exhibits the level of maturity I want.

**Comment by jaklan** [comment]
@klattimer I've actually pointed a few more issues here:
https://github.com/raycast/script-commands/tree/master/commands/remote-control/lg-tv

and thought about contributing to the project, but I've found out that library:
https://github.com/supersaiyanmode/PyWebOSTV

which generally has more functionalities and is quite well-written, but the only thing which is missing is CLI - so I've already started development of it. Imho it would be valuable to join your forces and maintain only one library, after adding CLI I don't see much value in existence of both of them.

**Comment by klattimer** [comment]
The pywebostv project was started by someone who didn't want to play with upstream development nicely. I've actually started adding the features for cursor control (the only feature above and beyond what I have here) but I'm also very busy so it's not a high priority. 

My intention with this project was to control my TV, I achieved that, and I'm happy to maintain this project going forward I also get a lot of activity here which is surprising. As far as integrations go, lgwebosremote is intended to be a library and a command line interface in it's distribution. 

I'm always happy to accept pull requests here though, so bare that in mind :)

For instance, if you were to create a fork with a scripts folder and import your scripts I would be happy to accept the pull :)
